sky/engine/core/StartUp.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/engine/core/StartUp.h"
```

Comment:
We use Google C++ style, so this file should be start_up.cc and the header should be start_up.h